docs/pages/integrations/abuseipdb/apis/check_ip.mdx

```diff
@@ -0,0 +1,83 @@
+import { Callout } from "nextra/components";
+
+# Check IP Address
+
+Retrieve detailed information about the reputation of a specific IP address.
+
+<Callout type="info">
+	For AbuseIPDB's documentation, see [CHECK
+	Endpoint](https://docs.abuseipdb.com/#check-endpoint).
+</Callout>
+
+**SDK Import:**
+
+```python
+from admyral.actions import abuseipdb_check_ip
+```
+
+## Arguments:
+
+| Argument Name                         | Description                                                                                      | Required |
+| ------------------------------------- | ------------------------------------------------------------------------------------------------ | :------: |
+| **IP Address** `ip_address`           | The IP address to check.                                                                         |   Yes    |
+| **Verbose** `verbose`                 | Include reports and the country name. Possible values: `yes`, `no`. Default: `no`.               |    -     |
+| **Max Age in Days** `max_age_in_days` | The maximum age in days of reports to consider. Values must be `>=1` and `<=365`. Default: `30`. |    -     |
+
+## Returns
+
+A JSON object.
+
+## Required Secrets
+
+| Secret Placeholder | Description                                                                |
+| ------------------ | -------------------------------------------------------------------------- |
+| `ABUSEIPDB_SECRET` | AbuseIPDB secret. See [AbuseIPDB setup](/integrations/abuseipdb/abuseipdb) |
+
+## SDK Example
+
+```python
+result = abuseipdb_check_ip(
+	ip_address="118.25.6.39",
+	verbose="yes",
+	max_age_in_days=90
+	secrets={
+		"ABUSEIPDB_SECRET": "my_stored_abuseipdb_secret"
+	}
+)
+```
+
+## Example Output:
+
+Typical successful response:
+
+```json
+{
+	"data": {
```

Comment:
when we return `response.json()["data"]`, please also update the example output.